### PR TITLE
implement autoslide and tests

### PR DESCRIPTION
**Pull request by cbikhazi**
New feature:  Able to implement an autoslide so that the slider automatically slides.  If property autoslide is set in milliseconds then it will autoslide otherwise the component will wait to UI.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "homepage": "https://github.com/erichbehrens/react-animated-slider#readme",
   "dependencies": {
+    "path": "^0.12.7",
```
Comment (erichbehrens) [inline package.json]: Where is this used?

Comment (cbikhazi) [reply package.json]: its not and it doesn't appear on my branch (not sure where it came from!), so if you want to remove then feel free.

**Comment by erichbehrens** [comment]
Thanks for your submission. I've implemented a few more things in #28 

- disable timer on mouseover/touch: don't you hate it when something slides away from under your mouse pointer? :D
- set timer recursively

Please note I've also changed `autoslide` prop to `autoplay` (more consistent with html's `<video />` element `autoplay` attribute.

**Comment by cbikhazi** [comment]
Hi,

Great stuff 😊

Have you updated the npm package?  I’d prefer to have a reference to your master code in the project I’m using the slider?

Thanks

Charles

**Pull request merged**